data/mods/blazemod/vehicleparts/blaze_weapons_parts.json

```diff
@@ -0,0 +1,993 @@
+[
```

Comment:
Did a couple of rudimentary changes here, though they are impactful. The `"copy-from":` field was changed from `blazemod`'s custom turrets to use vanilla precedent here- also, these are all from `blaze_autoweapons_parts.json`, the manual versions don't seem to be necessary because they all had auto equivalents. Duplicates that were essentially an "automated" version were removed, as that is all handled in vanilla with the` turret control unit` now. A smaller change here is that `"breaks_into":` was changed from `"count": [ 0, 1 ]` to the `"prob": 50` used in vanilla for these fields. Unsure about them breaking into their base component minus the turret, but for now that's left unchanged. Effectively they are unchanged, but this should be a considerable optimization of bloat code that was made obsolete by other PR's.